internal/zinc-core/src/main/scala/sbt/internal/inc/Relations.scala

```diff
@@ -184,13 +184,14 @@ trait Relations {
   private[inc] def externalDependencies: ExternalDependencies
 
   /**
-   * The class dependency relation between classes introduced by member reference.
+   * The class dependency relation between classes introduced by member reference excluding local references
```

Comment:
In Zinc, the term _local_ often means within the same subproject, i.e. local-to-Analysis, so "excluding same-source references" might be less ambiguous.